### DISCUSSION TITLE
API documentation

### DISCUSSION THREAD

**Discussion by made1990** [rejetto/hfs#385] [General]
Hey folks,

is there some kind of API specification or documentation for HFS?
Like OpenAPI (https://swagger.io/specification/)

That would be great.
Thx,
Marcus

**Comment by rejetto** [comment]
plugin api is documented, while the api used by web interfaces is currently undocumented.
Was you looking for something in particular?

**Reply by made1990** [reply]
Most intresting part is the http PUT command. What attributes need to be used/sent?

**Comment by rejetto** [comment]
please refer to this page
https://github.com/rejetto/hfs/wiki/Upload

**Reply by made1990** [reply]
Perfect, thx.
